Nodejs/Product/Nodejs/Options/NodejsNpmOptionsControl.cs

```diff
@@ -33,12 +33,14 @@ public partial class NodejsNpmOptionsControl : UserControl {
 
         internal void SyncControlWithPageSettings(NodejsNpmOptionsPage page) {
             _showOutputWhenRunningNpm.Checked = page.ShowOutputWindowWhenExecutingNpm;
+            _enableAutoTypeAcquisition.Checked = page.EnableAutomaticTypeAcquisition;
```

Comment:
Fixed the places I noticed this, but let me know if you see any other places that use the wrong names